.github/workflows/ci2.yml

```diff
@@ -137,6 +137,62 @@ jobs:
                             --build-arg DAOS_JAVA_BUILD=no
                             --build-arg DAOS_BUILD_TYPE=dev
                             --build-arg COMPILER
+  BuildARM:
+    name: Build DAOS (ARM64)
+    runs-on: [self-hosted, ARM64]
+    strategy:
+      fail-fast: false
+      matrix:
+        distro: [ubuntu]
+        compiler: [clang]
+        include:
+          - distro: ubuntu
+            base: ubuntu
+            with: ubuntu:22.04
```

Comment:
With only one element then you don't need a matrix here, unless there's a plan to add more probably easiest to do away with the matrix part.

Reply:
There is a plan to add more :) Just want to prove it stable first.